tutorial/part3/data_exploitability_pangeo.ipynb

```diff
@@ -2,9 +2,7 @@
  "cells": [
```

Comment:
<p>If you didn't test the time and have numbers I would suggest to add a statement that where you at least make users aware that in some cases apply Ufunc can be slower that the usual approach. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-data/pangeo-openeo-BiDS-2023/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>